videos/README.md

```diff
@@ -0,0 +1,50 @@
+# Video Workflow
+
+This document describes the components of the video workflow for OCW.
+
+**SECTIONS**
+
+1. [Overview](#overview)
+1. [Google Drive Sync and AWS Transcoding](#google-drive-sync-and-aws-transcoding)
+1. [YouTube Submission](#youtube-submission)
+1. [Captioning and 3Play Transcript Request](#captioning-and-3play-transcript-request)
+1. [Completing the Workflow](#completing-the-workflow)
+1. [Management Commands](#management-commands)
```

Comment:
It's amazing how GitHub renders the correct bullet numbers. I did not know this was possible, I've been doing unnecessary manual labor. 🤣